components/ItemForm.js

```diff
@@ -0,0 +1,133 @@
+import styled from "styled-components";
+import Link from "next/link";
+
+export default function ItemForm({ onSubmit }) {
+  function handleSubmit(event) {
+    event.preventDefault();
+    const data = Object.fromEntries(new FormData(event.target));
+
+    onSubmit(data);
+  }
+
+  return (
+    <Form onSubmit={handleSubmit}>
+      <CategoryWrapper>
+        <Label htmlFor="category-select">Category:</Label>
+        <Select name="category" id="category-select" required>
+          <option value="">Choose</option>
+          <option value="Lost">Lost</option>
+          <option value="Found">Found</option>
+        </Select>
+      </CategoryWrapper>
+      <Label htmlFor="user-name">User name:</Label>
+      <Input type="text" name="userName" id="user-name" required />
+      <Label htmlFor="title">Title of item:</Label>
+      <Input type="text" name="title" id="title" required />
+      <Label htmlFor="description">Description:</Label>
+      <Textarea
+        type="text"
+        name="description"
+        id="description"
+        rows="7"
+        required
+      />
+      <ButtonsWrapper>
+        <BackLink href={"/"} aria-label="cancel">
```

Comment:
your link should also work without the brackets

```suggestion
        <BackLink href="/" aria-label="cancel">
```